source/playbooks/SC/ssmdocs/SC_AutoScaling.1.ts

```diff
@@ -17,7 +17,7 @@ class EnableAutoScalingGroupELBHealthCheckDocument extends ControlRunbookDocumen
       remediationName: 'EnableAutoScalingGroupELBHealthCheck',
       scope: RemediationScope.REGIONAL,
       resourceIdName: 'AutoScalingGroupName',
-      resourceIdRegex: String.raw`^arn:(?:aws|aws-cn|aws-us-gov):autoscaling:(?:[a-z]{2}(?:-gov)?-[a-z]+-\d):\d{12}:autoScalingGroup:(?:[0-9a-fA-F]{11}-(?:[0-9a-fA-F]{4}-){3}[0-9a-fA-F]{12}):autoScalingGroupName\/(.*)$`,
+      resourceIdRegex: String.raw`^arn:(?:aws|aws-cn|aws-us-gov):autoscaling:(?:[a-z]{2}(?:-gov)?-[a-z]+-\d):\d{12}:autoScalingGroup:(?:[0-9a-fA-F]{8}-(?:[0-9a-fA-F]{4}-){3}[0-9a-fA-F]{12}):autoScalingGroupName/(.{1,255})$`,
```

Comment:
This is a fix we made in v1.5.1 to the other standards but it didn't get merged forward. I reviewed all the CRs between v1.5.0 and v1.5.1 for anything similar and found nothing.

This does bring up the fact that the regexes in SC are not validated against the regex registry. I'll add an item to do that.